code/src/components/Header.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Route, Link } from 'react-router-dom'
+
+import { ReactComponent as Mustache } from 'media/mustache.svg'
+
+import './Header.css'
+
+export const Header = () => {
+  return (
+    <header>
+      <h1 className='page-title'>Epic mustaches movies list!</h1>
+      <Route path='/movies'>
+      <Link to={"/"}>
+      <span className='mustache-icon'><Mustache /><p className='go-back'>Go back to movies</p></span>
+        </Link>
+      </Route>
```

Comment:
I like that you have a header and that you using this method to decide that some content shall only be presented at `<Route path='/movies'> `